v3/plugins/redhat/vscode-wsdl2rest/latest/meta.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
+publisher: redhat
+name: vscode-wsdl2rest
+version: latest
+type: VS Code extension
+displayName: wsdl2rest support for Apache Camel
+title: wsdl2rest support for Apache Camel
+description: Extension supporting the mapping of an existing SOAP service to a RESTful service in Camel Rest DSL generated via wsdl2rest.
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/camel-tooling/vscode-wsdl2rest
+category: Other
+firstPublicationDate: "2019-02-26"
+spec:
+  containers:
+    - image: "eclipse/che-remote-plugin-runner-java8:next"
+      memoryLimit: "1000Mi"
```

Comment:
Is this a minimum requirement for memory? With a requirement of `1000Mi`, running a workspace with this plugin and the Java LS will already consume over `3Gi`. 

From an OSIO perspective, it would be great if this default limit could be lowered.

Reply:
Other than thru trial and error is there a way for us to determine how much memory is actually needed @amisevsk ? Or do we even need that container? I was just going by what was being done by other extensions

Reply:
@bfitzpat trial and error is thus far the best approach I've found; I usually open a terminal and monitor memory usage while trying to tax the plugin (e.g. for Java LS, view mem usage while its importing the project). From there I adjust up a little.

As for whether the container is needed, I would ask one of our theia/plugin devs. I can't confidently say one way or another. Some plugins don't require sidecars, e.g. the github pull request plugin.